src/screens/monitorRequests/components/MonitorsRequestListItem/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { Typography } from '@mui/material';
+import { Button } from '../../../../components/button';
+import { TMonitorRequest } from '../../../../service/requests/useGetAllMonitorRequests/types';
+import { TypeUserEnum } from '../../../../utils/constants';
+import theme from '../../../../utils/theme';
+import { ButtonsContainer, Container, DataContainer } from './styles';
+
+type Props = {
+  request: TMonitorRequest;
+  userTypeId?: number;
+  handleChangeCurrentMonitorAccept(monitor: TMonitorRequest): void;
+  handleOpenAcceptModal(): void;
+  handleChangeCurrentMonitorDeny(monitor: TMonitorRequest): void;
+  handleOpenDenyModal(): void;
+};
+
+const MonitorsRequestListItem = ({
+  request,
+  userTypeId,
+  handleChangeCurrentMonitorAccept,
+  handleOpenAcceptModal,
+  handleChangeCurrentMonitorDeny,
+  handleOpenDenyModal,
+}: Props) => {
+  return (
+    <Container>
+      <DataContainer>
+        <Typography
+          textAlign={'center'}
+          style={{ color: `${theme.palette.grey[600]}` }}
+          variant="caption"
+        >
+          {request.student.course.name}
+        </Typography>
+        <Typography
+          textAlign={'center'}
+          style={{ margin: '4px 0' }}
+          variant="h6"
+        >
+          {request.student.user.name}
+        </Typography>
+        <Typography
+          textAlign={'center'}
+          style={{ color: `${theme.palette.grey[600]}`, margin: '12px 0' }}
+          variant="caption"
+        >
+          Quer ser monitor da disciplina
+        </Typography>
+        <Typography
+          textAlign={'center'}
+          style={{ margin: '6px 0' }}
+          variant="subtitle1"
+        >
+          {request.subject.name}
+        </Typography>
+        {userTypeId === TypeUserEnum.COORDINATOR ? (
+          <Typography
+            textAlign={'center'}
+            style={{ color: `${theme.palette.grey[600]}` }}
+            variant="caption"
+          >
+            Prof. {request.responsible_professor.user.name}
+          </Typography>
+        ) : null}
+        <ButtonsContainer>
+          <Button
+            onClick={() => {
+              handleChangeCurrentMonitorDeny(request);
+              handleOpenDenyModal();
```

Comment:
Por que você separou essas duas funções? Podia ser só uma:
```suggestion
              handleOpenDenyModal(request);
```

Reply:
Feito